templates/index.html

```diff
@@ -110,12 +121,34 @@ <h3 class="p-heading--2">Hybrid, multi-cloud <br class="u-hide--medium u-hide--s
         <div class="row">
           <div class="col-6">
             <h3 class="p-heading--2">Optimised <br class="u-hide--medium u-hide--small">for IoT and edge</h3>
-            <p>Speed up time-to-market and lower maintenance costs. Ship devices that are self-healing, secure and always up-to-date.</p>
+            <p>Speed up time-to-market and lower maintenance costs. Ship devices that are self-healing, secure and
+              always up-to-date.</p>
           </div>
           <div class="col-3">
             <p class="p-heading--2"><a href="https://ubuntu.com/core">Ubuntu Core&nbsp;&rsaquo;</a></p>
             <hr class="p-rule is-dark u-hide--medium">
-            <p class="p-heading--2"><a href="https://microk8s.io/">MicroK8s&nbsp;&rsaquo;</a><p>
+            <p class="p-heading--2"><a href="https://microk8s.io/">MicroK8s&nbsp;&rsaquo;</a>
+            <p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+  <section class="p-section">
+    <div class="row">
+      <div class="col-start-medium-3 col-medium-4 col-start-large-4 col-9">
+        <hr class="p-rule is-dark">
+        <div class="row">
+          <div class="col-6">
+            <h3 class="p-heading--2">Multi-cloud data fabric <br class="u-hide--medium u-hide--small">and AI</h3>
+            <p>Focus more on innovation and simplify operations with our multi-cloud data and AI solutions. Get the 
+              predictable economics you need to scale your projects.</p>
+          </div>
+          <div class="col-3">
+            <p class="p-heading--2"><a href="/data">Data Fabric&nbsp;&rsaquo;</a></p>
```

Comment:
href should probably be `https://ubuntu.com/data`

Reply:
nope, this one is correct. We are removing data bubble from ubuntu.com and creating it in canonical.com

Reply:
Pinging @Sophie-32 as it's a copy matter.